packages/handler-graphql/src/responses.ts

```diff
@@ -17,13 +17,39 @@ export class ErrorResponse {
         message: string;
         data?: any;
     };
-    constructor(params: ErrorResponseParams) {
+    constructor(params: ErrorResponseParams | Error | (Error & ErrorResponseParams)) {
         this.data = null;
-        this.error = {
-            code: params.code || defaultParams.code,
-            message: params.message || defaultParams.message,
-            data: params.data || defaultParams.data
-        };
+
+        if (params instanceof Error) {
+            // usually we don't use env variables directly inside code
+            // but otherwise it would make it very tedious to pass this through multiple layers of code
+            const debug = process.env.DEBUG === "true";
+
+            this.error = {
+                code: defaultParams.code,
```

Comment:
I simplified it, but couldn't remove `"code" in params` check, because of TS typings. Error can be a simple `Error` instance that does not have `code` or `data` props. I could make type casting to `any` but it's not pretty.